content/workload-reduction-toolkit/how-the-toolkit-was-made.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How the toolkit was made and tested
+---
+
+# How the toolkit was made and tested
+
+The toolkit is a set of workload reduction resources produced by school leaders, for school leaders. They have been
+designed to support what you are already doing and can be adapted to work in your school.
+
+---
```

Comment:
Not sure how easy this is to fix, but this section break is running across the whole of the page:

<img width="1261" alt="Screenshot 2024-01-15 at 11 33 25" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/e42f90f5-43d3-4da0-9836-46c9ee086761">

In the designs it's contained within the same width container as the text.

Reply:
I applied some CSS 👍